bliss.shy.js

```diff
@@ -54,16 +54,15 @@ extend($, {
 	 * Returns the [[Class]] of an object in lowercase (eg. array, date, regexp, string etc)
 	 */
 	type: function(obj) {
-		if (obj === null) { return 'null'; }
-
-		if (obj === undefined) { return 'undefined'; }
-
-		var ret = (Object.prototype.toString.call(obj).match(/^\[object\s+(.*?)\]$/)[1] || "").toLowerCase();
-
-		if(ret == 'number' && isNaN(obj)) {
-			return 'nan';
+		var ret;
+		if(obj === null || obj === undefined) {
+			ret = obj + '';
+		} else {
```

Comment:
Please follow the code style of the rest of the code. `else {` should be on a separate line.

@dperrymorrow is there a way to enforce this via jshint?